packages/babel-plugin-glaze/src/index.ts

```diff
@@ -0,0 +1,305 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/* eslint-disable no-param-reassign */
+import { NodePath, PluginObj, template, types } from '@babel/core';
```

Comment:
I had to turn off some of the eslint lint for this file. Was a bit lazy to properly handle them and actually I'm not sure if that makes sense to enable them for this file...